index.html

```diff
@@ -0,0 +1,57 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta name="Author" content="Shubh M Das">
+    <link rel="stylesheet" href="style.css">
+    <title>portfolio</title>
+</head>
+
+<body>
+   <header>
+        <div class="container">
+            <div id="branding">
+                <h3>My Logo</h3>
+            </div>
+            <nav>
+                <ul>
+                 <li><a href="#">Portfolio</a></li> 
+                 <li><a href="#">About</a></li>
+                 <li><a href="#">Contact</a></li>
+                </ul>
+            </nav>
+
+        </div>
+   </header>
+
+   <section class="showcase">
+       <div class="image-1">
+           <img src="./images/ellipse11.svg" alt="styling image">
+           <img src="./images/ellipse7.svg" alt="styling image">
+           <img src="./images/ellipse3.svg" alt="styling image">
+       </div>
+       <div class="hero">
+       <h1>I'm Shubh</h1>
+       <h1>Glad to see you!</h1>
+       <br>
+       <p>I’m a software developer! I can help you build a product,
+            feature or website Look through some of my work and experience! <br>
+            If you like what you see and have a project you need coded, don’t 
+            hestiate to contact me.</p>
+            <br>
+            <p class="connect">LET'S CONNECT</p>
+            <div class="social-icon">
+                <img src="./images/linkedin.svg" alt="linkedin">
+                <img src="./images/github.svg" alt="github">
+                <img src="./images/angellist.svg" alt="angellist">
+            </div> 
+            <div class="image-2">
```

Comment:
- [ ] Please, consider, that the list of icons uses a `ul` tag to group them, and `li` per icon; each icon has a link (`a` tag)